.swiftpm/xcode/xcshareddata/xcschemes/BrowserServicesKit-Package.xcscheme

```diff
@@ -722,6 +736,16 @@
                ReferencedContainer = "container:">
             </BuildableReference>
          </TestableReference>
+         <TestableReference
+            skipped = "NO">
+            <BuildableReference
+               BuildableIdentifier = "primary"
+               BlueprintIdentifier = "PhishingDetectionTests"
```

Comment:
should this be here?

Reply:
Manually removed.   These were not my changes.   There were some changes in main branch when I branched out, and then these were reverted.